packages/x-date-pickers-pro/src/DateRangeCalendar/DateRangeCalendar.tsx

```diff
@@ -581,20 +577,23 @@ const DateRangeCalendar = React.forwardRef(function DateRangeCalendar(
               {...baseDateValidationProps}
               {...commonViewProps}
               onMonthSwitchingAnimationEnd={onMonthSwitchingAnimationEnd}
-              onFocusedDayChange={changeFocusedDay}
+              onFocusedDayChange={(focusedDate) =>
```

Comment:
Nitpick: currently all the individual props we pass to the component are stable (their references don't change), have you considered keeping it like this, even though it will not reduce the amount of re-rendering? 😆

Reply:
I missed that one sorry
If your OK with this I won't spend time on cleaning the component too much since we want to migrate away from it (and on to the new Base UI implementation) in the coming months :crossed_fingers: